UI/data/locale/en-US.ini

```diff
@@ -1030,3 +1034,19 @@ XSplitBroadcaster="XSplit Broadcaster"
 # OBS restart
 Restart="Restart"
 NeedsRestart="OBS Studio needs to be restarted. Do you want to restart now?"
+
+# Context Bar
+ContextBar.NoSelectedSource="No source selected"
+ContextBar.ResetTransform="Reset Transform"
+ContextBar.FitToCanvas="Fit to Canvas"
+ContextBar.CloseContextToolbar="Close Context Toolbar"
```

Comment:
Other text calls this the "Source Toolbar" - this text should be updated to "Close Source Toolbar"